libkbfs/folder_block_ops.go

```diff
@@ -1002,17 +1007,31 @@ func (fbo *folderBlockOps) RemoveDirEntryInCache(lState *lockState, dir path,
 // longer needed.
 func (fbo *folderBlockOps) RenameDirEntryInCache(lState *lockState,
 	oldParent path, oldName string, newParent path, newName string,
-	newDe DirEntry, replacedDe DirEntry) dirCacheUndoFn {
+	newDe DirEntry, replacedDe DirEntry) (dirCacheUndoFn, error) {
 	fbo.blockLock.Lock(lState)
 	defer fbo.blockLock.Unlock(lState)
 	if newParent.tailPointer() == oldParent.tailPointer() &&
 		oldName == newName {
 		// Noop
-		return fbo.wrapWithBlockLock(func() {})
+		return fbo.wrapWithBlockLock(func() {}), nil
```

Comment:
I'll be changing this to just return `nil` in the future, and checking it at the caller.